cx_Freeze/finder.py

```diff
@@ -218,7 +218,7 @@ def _FindModule(self, name, path, namespace):
                 # Namespace package (?)
                 module = sys.modules[name]
                 info = ("", "", imp.PKG_DIRECTORY)
-                return None, module.__path__[0], info
+                return None, next(p for p in module.__path__), info
```

Comment:
Can you provide the script that causes this problem to occur? I'd like to take a closer look. Thanks.